docker/build_scripts/build-git.sh

```diff
@@ -7,6 +7,10 @@ set -exuo pipefail
 # Get script directory
 MY_DIR=$(dirname "${BASH_SOURCE[0]}")
 
+if [ "${POLICY}" == "musllinux_1_1" ]; then 
+    export NO_REGEX=yes
```

Comment:
It would be great not to have this exception for musllinux. Is there a build dependency missing or something else ?

Reply:
Unfortunately git requires REG_STARTEND which is [not available in musl libc](https://www.openwall.com/lists/musl/2013/01/15/14).

Reply:
thanks for the reference. Given the commit in git, using
```suggestion
    export NO_REGEX=NeedsStartEnd
```
might help to find that reference later on.